neofs-testlib/neofs_testlib_tests/pytest/test_env.py

```diff
@@ -205,7 +206,7 @@ def test_node_metabase_resync(neofs_env: NeoFSEnv, wallet: NodeWallet, zero_fee)
     for node in neofs_env.storage_nodes:
         node.set_metabase_resync(True)
         node.set_metabase_resync(False)
-    test_http_gw_put_get(neofs_env, wallet, zero_fee)
+    test_gateways_put_get(neofs_env, wallet, zero_fee, "HTTP")
```

Comment:
This doesn't look like a linter/formatter change.

Reply:
It actually was shown by the linter. Original method signature was changed sometimes ago, but the corresponding usages were not. Linter showed that there is no test_http_gw_put_get method.

Reply:
But I will move it to a separate commit.